src/SFA.DAS.EmployerIncentives.Web/Controllers/ApplyController.cs

```diff
@@ -48,7 +56,19 @@ public async Task<IActionResult> SubmitApplication(string accountId, Guid applic
 
             await _applicationService.Confirm(accountId, applicationId, email, string.Join(" ", firstName, lastName));
 
-            return RedirectToAction("BankDetailsConfirmation", "BankDetails", new { accountId, applicationId });
```

Comment:
Wouldn't it be better to redirect to the BankDetailsConfirmation as is and put the VRF check in that method since it's a convern of the bankdetails controller.